docs/framework/development-tools/package-generator/javascript-package.md

```diff
@@ -124,7 +130,22 @@ Examples:
 npm run stylelint
 ```
 
-### `dll:build`
+### `build:dist`
+
+Creates npm and browser builds of your plugin. These builds can be added to the editor by following the TODO: LINK/NEW GUIDE??.
```

Comment:
Wouldn't that be link to the legacy methods?

Reply:
We already have a migration guide placeholder, so in that case we can safely link it: `{@link updating/new-installation-methods Migrating CKEditor 5 to new installation methods} guide`.

Reply:
Adressed in 97d00c4.